Cython/Includes/libcpp/vector.pxd

```diff
@@ -113,20 +113,20 @@ cdef extern from "<vector>" namespace "std" nogil:
             bint operator>=(const_reverse_iterator)
 
         vector() except +
-        vector(vector&) except +
+        vector(const vector&) except +
         vector(size_type) except +
-        vector(size_type, T&) except +
-        #vector[InputIt](InputIt, InputIt)
+        vector(size_type, const T&) except +
+        vector(iterator, iterator) except +
```

Comment:
TODO: if templating - add fake type. Decision pending.

Reply:
This should be templated I think because the constructor is for any input iterators rather than just the std::vector ones